app/views/account/details.scala.html

```diff
@@ -52,7 +52,7 @@ <h1 class="suspend-header__title">Manage your Guardian subscription</h1>
                             </legend>
                             <div class="field-panel__intro">
                                 <div class="field-note field-note--offset">
-                                @fragments.forms.securityNote()
+                                @_root_.views.html.fragments.forms.securityNote()
```

Comment:
the imports in play 2.6 seems to work differently, so it finds the fragments package in the current directory, rather than the root one.  I've explicitly specified it in the affected places and it seems to get around it.